tests/clients/test_ngsi_ld_cb.py

```diff
@@ -91,28 +91,38 @@ def test_statistics(self):
         """
         self.assertIsNotNone(self.client.get_statistics())
 
-    def aatest_pagination(self):
+    def test_pagination(self):
         """
         Test pagination of context broker client
         Test pagination. only works if enough entities are available
-        """
-        fiware_header = FiwareLDHeader()
-        with ContextBrokerLDClient(fiware_header=fiware_header) as client:
-            entities_a = [ContextLDEntity(id=f"urn:ngsi-ld:test:{str(i)}",
-                                        type=f'filip:object:TypeA') for i in
-                          range(0, 1000)]
-            client.entity_batch_operation(action_type=ActionTypeLD.CREATE, entities=entities_a)
-            entities_b = [ContextLDEntity(id=f"urn:ngsi-ld:test:{str(i)}",
-                                        type=f'filip:object:TypeB') for i in
-                          range(1000, 2001)]
-            client.entity_batch_operation(action_type=ActionTypeLD.CREATE, entities=entities_b)
-            self.assertLessEqual(len(client.get_entity_list(limit=1)), 1)
-            self.assertLessEqual(len(client.get_entity_list(limit=999)), 999)
-            self.assertLessEqual(len(client.get_entity_list(limit=1001)), 1001)
-            self.assertLessEqual(len(client.get_entity_list(limit=2001)), 2001)
-
-            client.entity_batch_operation(action_type=ActionTypeLD.DELETE, entities=entities_a)
-            client.entity_batch_operation(action_type=ActionTypeLD.DELETE, entities=entities_b)
+        self.assertLessEqual(len(self.client.get_entity_list(limit=1)), 1)
```

Comment:
I put that on the side, in case someone has some other intentions with it. You can remove it if you think it is not needed. The re-writing part in that test had to do with it failing for certain batch sizes and thats it... beyond that, im not really certain when exactly this happens